datamade/la-metro-councilmatic
# Minutes Page

## Overview

This PR adds a Minutes page to the navigation. It incorporates some historical events from a csv file in the Minutes page table and includes a UI to filter the events by date.

### Checklist


## Testing Instructions

 * Visit the staging site and click on the new "Minutes" link in the nav bar
 * Make sure the new Minutes page looks good. Try using the date filter and all the buttons associated with that filter

Handles #770
